[
  {
    "project": "canonical/subiquity",
    "commit": "94de120d16eccf1075d9c96ed6654aa5d3b8094d",
    "filename": "subiquity/common/errorreport.py",
    "filename_after_commit": "/home/chowyi/TypeAnnotation_Study/GitHub/canonical-subiquity/subiquity/common/errorreport.py",
    "file_hunks_size": 10,
    "min_patch_found": false,
    "single_hunk": false,
    "fit_TFix": false,
    "delete_only_patch": false,
    "has_suppression_all_hunks": false,
    "full_warning_msg": "subiquity/common/errorreport.py:55:23 Unbound name [10]: Name `_` is used but not defined in the current scope.",
    "message": " Name `_` is used but not defined in the current scope.",
    "rule_id": "Unbound name [10]",
    "warning_line_no": 55,
    "warning_line": "    BLOCK_PROBE_FAIL = _(\"Block device probe failure\")"
  }
]